prime-router/src/main/kotlin/cli/LookupTableCommands.kt

```diff
@@ -844,7 +897,7 @@ class LookupTableLoadAllCommand : GenericLookupTableCommand(
             if (needToLoad) {
                 TermUi.echo("Creating table $tableName...")
                 val args = mutableListOf(
-                    "-e", environment.toString().lowercase(), "-n", tableName,
+                    "-e", environment.toString().lowercase(), "-n", tableName, "-c", tableSha256,
```

Comment:
Don't let a user pass a checksum.  Yes, that will mean you may need to compute the checksum twice.  Check in your tests to see how slow this gets.

Reply:
Done.